embeds/linux_icon.go

```diff
@@ -0,0 +1,62 @@
+package embeds
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/anywherelan/awl/config"
+)
+
+const (
+	awlDirPath  = ".local/share/Anywherelan"
```

Comment:
Is this necessary to have a subdir `Anywherelan` in `share`?  Can we have icon in `~/.local/share/icons/` or this path is deprecated?

Reply:
icons dir is ok, I copy this way of usage from Jetbrains soft. Will fix it